csi/start.py

```diff
@@ -10,6 +10,7 @@ def main():
     mon = Monitor()
     mon.add_process(Proc("csi", "python3", [curr_dir + "/main.py"]))
     mon.add_process(Proc("metrics", "python3", [curr_dir + "/exporter.py"]))
+    mon.add_process(Proc("quota", "bash", [curr_dir + "/quota-crawler.sh"]))
```

Comment:
Same container image is pulled by `csi-provisioner` and `csi-nodeplugin`. To avoid starting this process in `nodeplugin`, we can add one env variable in csi.yaml.j2 template(Ex: `CSI_ROLE=provisioner`).

```
if os.environ.get("CSI_ROLE", "-") == "provisioner":
    mon.add_process(Proc("quota", "bash", [curr_dir + "/quota-crawler.sh"]))
```